src/cropareaest_funcs.py

```diff
@@ -0,0 +1,564 @@
+# Import libraries
+import os
+import pandas as pd
+import numpy as np
+import geopandas as gpd
+import rasterio as rio
+from rasterio.mask import mask
+import cartopy.io.shapereader as shpreader
+from shapely.geometry import box
+from sklearn.metrics import confusion_matrix
+
+
+def return_map_bounds_as_df():
+    "Read the bounding coordinates of the crop mask"
+    root_dir = os.getcwd()
+    prj_raster = [f for f in os.listdir(root_dir) if f.startswith("prj")][0]
+
+    with rio.open(prj_raster) as src:
+        bbx = src.bounds
+    geom = box(*bbx)
+    bbx_gdf = gpd.GeoDataFrame(geometry=[geom], crs=src.crs)
+    return bbx_gdf
```

Comment:
nit: I think this could just be one line
```python
with rio.open(prj_raster) as src:
    return gpd.GeoDataFrame(geometry=[box(*src.bounds)], crs=src.crs)
```